src/mlpack/tests/io_test.cpp

```diff
@@ -868,7 +889,8 @@ BOOST_AUTO_TEST_CASE(UnmappedParamTest)
  * Test that we can serialize a model and then deserialize it through the CLI
  * interface.
  */
-BOOST_AUTO_TEST_CASE(SerializationTest)
+TEST_CASE_METHOD(IOTestDestroyer, "IOSerializationTest",
+                "[IOTest]")
```

Comment:
```suggestion
TEST_CASE_METHOD(IOTestDestroyer, "IOSerializationTest", "[IOTest]")
```

Looks like this would fit into the same line.